.devcontainer/devcontainer.json

```diff
@@ -23,58 +23,59 @@
 		"type=bind,source=/var/run/docker.sock,target=/var/run/docker.sock"
 	],
 
-	// Set *default* container specific settings.json values on container create.
-	"settings": { 
-		"python.pythonPath": "/opt/conda/envs/development/bin/python",
-		"python.languageServer": "Pylance",
-		"python.linting.enabled": true,
-		"python.linting.pylintEnabled": true,
-		"python.formatting.autopep8Path": "/usr/local/py-utils/bin/autopep8",
-		"python.formatting.blackPath": "/usr/local/py-utils/bin/black",
-		"python.formatting.yapfPath": "/usr/local/py-utils/bin/yapf",
-		"python.linting.banditPath": "/usr/local/py-utils/bin/bandit",
-		"python.linting.flake8Path": "/usr/local/py-utils/bin/flake8",
-		"python.linting.mypyPath": "/usr/local/py-utils/bin/mypy",
-		"python.linting.pycodestylePath": "/usr/local/py-utils/bin/pycodestyle",
-		"python.linting.pydocstylePath": "/usr/local/py-utils/bin/pydocstyle",
-		"python.linting.pylintPath": "/usr/local/py-utils/bin/pylint",
-        "files.watcherExclude": {
-            "**/.git/objects/**": true,
-            "**/.git/subtree-cache/**": true,
-            "**/node_modules/*/**": true,
-            "**/.python_packages/*/**": true
-          },
-		"files.associations": {
-			"*.workbook": "[jsonc]"
+	// Set *default* container specific settings values on container create.
+	"customizations": {
+		"vscode": {
+			"settings": { 
```

Comment:
I am still getting the following warnings after rebuilding my devconainer with these changes... 
![image](https://github.com/microsoft/PubSec-Info-Assistant/assets/48474707/af8ec627-eb34-4190-8ee6-887388b0d220)

```
2024-03-01 13:11:06.262 [info] Experiment 'pythonRecommendTensorboardExt' is active
2024-03-01 13:11:06.262 [info] Experiment 'pythonREPLSmartSend' is active
2024-03-01 13:11:06.262 [info] Experiment 'pythonSurveyNotification' is active
2024-03-01 13:11:06.262 [info] Experiment 'pythonTerminalEnvVarActivation' is active
2024-03-01 13:11:06.262 [info] Experiment 'pythonTestAdapter' is active
2024-03-01 13:11:06.262 [info] Default formatter is set to null for workspace /workspaces/info-asst
2024-03-01 13:11:06.262 [error] Following setting is deprecated: "python.linting.pylintEnabled"
2024-03-01 13:11:06.262 [error] All settings starting with "python.linting." are deprecated and can be removed from settings.
2024-03-01 13:11:06.262 [error] Linting features have been moved to separate linter extensions.
2024-03-01 13:11:06.262 [error] See here for more information: https://code.visualstudio.com/docs/python/linting
2024-03-01 13:11:06.262 [error] Please install "pylint" extension: https://marketplace.visualstudio.com/items?itemName=ms-python.pylint
2024-03-01 13:11:06.262 [info] Test server listening.
2024-03-01 13:11:06.449 [warning] Retry failed, storage update failed for key PYTHON_GLOBAL_STORAGE_KEYS
2024-03-01 13:11:06.484 [warning] Retry failed, storage update failed for key PYTHON_GLOBAL_STORAGE_KEYS
2024-03-01 13:11:06.484 [warning] Retry failed, storage update failed for key PYTHON_GLOBAL_STORAGE_KEYS
2024-03-01 13:11:06.484 [warning] Retry failed, storage update failed for key PYTHON_GLOBAL_STORAGE_KEYS
2024-03-01 13:11:06.484 [warning] Retry failed, storage update failed for key PYTHON_GLOBAL_STORAGE_KEYS
2024-03-01 13:11:06.484 [warning] Retry failed, storage update failed for key PYTHON_GLOBAL_STORAGE_KEYS
2024-03-01 13:11:06.484 [warning] Retry failed, storage update failed for key PYTHON_GLOBAL_STORAGE_KEYS
2024-03-01 13:11:06.484 [warning] Retry failed, storage update failed for key PYTHON_GLOBAL_STORAGE_KEYS
2024-03-01 13:11:06.484 [warning] Retry failed, storage update failed for key PYTHON_GLOBAL_STORAGE_KEYS
2024-03-01 13:11:06.484 [warning] Retry failed, storage update failed for key PYTHON_GLOBAL_STORAGE_KEYS
2024-03-01 13:11:06.534 [info] > conda info --json
2024-03-01 13:11:06.549 [warning] Retry failed, storage update failed for key PYTHON_GLOBAL_STORAGE_KEYS
2024-03-01 13:11:06.549 [warning] Retry failed, storage update failed for key PYTHON_GLOBAL_STORAGE_KEYS
2024-03-01 13:11:06.549 [warning] Retry failed, storage update failed for key PYTHON_GLOBAL_STORAGE_KEYS
2024-03-01 13:11:06.586 [info] > /usr/bin/python3 -I ~/.vscode-server/extensions/ms-python.python-2024.2.1/pythonFiles/get_output_via_markers.py ~/.vscode-server/extensions/ms-python.python-2024.2.1/pythonFiles/interpreterInfo.py
2024-03-01 13:11:06.610 [info] > /usr/local/bin/python -I ~/.vscode-server/extensions/ms-python.python-2024.2.1/pythonFiles/get_output_via_markers.py ~/.vscode-server/extensions/ms-python.python-2024.2.1/pythonFiles/interpreterInfo.py
2024-03-01 13:11:06.643 [info] > ./.venv/bin/python -I ~/.vscode-server/extensions/ms-python.python-2024.2.1/pythonFiles/get_output_via_markers.py ~/.vscode-server/extensions/ms-python.python-2024.2.1/pythonFiles/interpreterInfo.py
2024-03-01 13:11:06.678 [info] Python interpreter path: ./.venv/bin/python
2024-03-01 13:11:06.794 [warning] Retry failed, storage update failed for key PYTHON_ENV_INFO_CACHE
2024-03-01 13:11:06.794 [warning] Retry failed, storage update failed for key PYTHON_ENV_INFO_CACHE
2024-03-01 13:11:06.911 [info] > . ./.venv/bin/activate && echo 'e8b39361-0157-4923-80e1-22d70d46dee6' && python ~/.vscode-server/extensions/ms-python.python-2024.2.1/pythonFiles/printEnvVariables.py
2024-03-01 13:11:06.911 [info] shell: bash
2024-03-01 13:11:06.913 [warning] Retry failed, storage update failed for key PYTHON_ENV_INFO_CACHE
2024-03-01 13:11:06.913 [warning] Retry failed, storage update failed for key PYTHON_ENV_INFO_CACHE
2024-03-01 13:11:06.999 [info] > . ./.venv/bin/activate && echo 'e8b39361-0157-4923-80e1-22d70d46dee6' && python ~/.vscode-server/extensions/ms-python.python-2024.2.1/pythonFiles/printEnvVariables.py
2024-03-01 13:11:06.999 [info] shell: bash
2024-03-01 13:11:07.007 [warning] Retry failed, storage update failed for key PYTHON_ENV_INFO_CACHE
2024-03-01 13:11:07.049 [info] > /usr/bin/python3 ~/.vscode-server/extensions/ms-python.python-2024.2.1/pythonFiles/printEnvVariables.py
2024-03-01 13:11:07.049 [info] shell: bash
2024-03-01 13:11:07.070 [info] Send text to terminal: /usr/bin/python3 /home/vscode/.vscode-server/extensions/ms-python.python-2024.2.1/pythonFiles/printEnvVariablesToFile.py /home/vscode/.vscode-server/extensions/ms-python.python-2024.2.1/pythonFiles/deactivate/bash/envVars.txt
2024-03-01 13:11:08.030 [info] Starting Pylance language server.
```

Reply:
Partly my fault, I told him I didn't think we needed to update the settings. But i thought the ask was just python and extensions checking. Sorry Jeff

Reply:
NP Ryon, I'll test locally with rebuild without cache to duplicate this issue. Thanks for bringing this to my attention Daylan